source/tools/environments.rst

```diff
@@ -0,0 +1,54 @@
+Environments
+============
+
+.. role:: bash(code)
+   :language: bash
+
+*Why?*
+
+     What is an environment? Why should you use one?
+
+     Most of the libraries we use and the programs we code are made in python. But also, most of the programs in Linux are based on python. We want to make sure that when we work, install libraries, uninstall libraries, etc., **we don’t break your Linux installation!** Environments are a good way to separate the system's python installation from the python version you need for your work.
+
+     For instance, a python 3 environment will be necessary to use scilpy (the public repository) or Dipy1.0, but you might need a python 2 environment to work on our private repository of scilpy.
+
+*What is it?*
+
+    There are many options for environments. One very famous option is the Anaconda/conda world, but in the lab, we have experienced some issues with it. We highly recommand using `VirtualEnv <https://virtualenv.pypa.io/en/latest/>`_.
+
+    .. code-block:: bash
+
+        sudo pip install virtualenvwrapper
+        mkdir ~/Envs
+        NAME=somename # Ex: $NOM could be my_env_py3
+                      # depending on the use you will give to this environment.
+                      # Try to be explicit
+        # One of:
+        mkvirtualenv $NAME
+        mkvirtualenv --python=python3.7 $NAME   # To use one precise python version
```

Comment:
Are you sure this line works ? I think --python take only a path

Reply:
I just checked and it works!